board.go

```diff
@@ -111,24 +111,33 @@ type (
 		Value interface{}        `json:"value"` // TODO select more precise type
 	}
 	Annotation struct {
-		Name        string         `json:"name"`
-		Datasource  *DatasourceRef `json:"datasource"`
-		ShowLine    bool           `json:"showLine"`
-		IconColor   string         `json:"iconColor"`
-		LineColor   string         `json:"lineColor"`
-		IconSize    uint           `json:"iconSize"`
-		Enable      bool           `json:"enable"`
-		Query       string         `json:"query"`
-		Expr        string         `json:"expr"`
-		Step        string         `json:"step"`
-		TextField   string         `json:"textField"`
-		TextFormat  string         `json:"textFormat"`
-		TitleFormat string         `json:"titleFormat"`
-		TagsField   string         `json:"tagsField"`
-		Tags        []string       `json:"tags"`
-		TagKeys     string         `json:"tagKeys"`
-		Type        string         `json:"type"`
+		Name        string             `json:"name"`
+		Datasource  *DatasourceRef     `json:"datasource"`
+		ShowLine    bool               `json:"showLine"`
+		IconColor   string             `json:"iconColor"`
+		LineColor   string             `json:"lineColor"`
+		IconSize    uint               `json:"iconSize"`
+		Enable      bool               `json:"enable"`
+		Query       string             `json:"query"`
+		Expr        string             `json:"expr"`
+		Step        string             `json:"step"`
+		TextField   string             `json:"textField"`
+		TextFormat  string             `json:"textFormat"`
+		TitleFormat string             `json:"titleFormat"`
+		TagsField   string             `json:"tagsField"`
+		Tags        []string           `json:"tags"`
+		TagKeys     string             `json:"tagKeys"`
+		Type        string             `json:"type"`
+		Filter      []AnnotationFilter `json:"filter"`
```

Comment:
```suggestion
		Filter      AnnotationFilter `json:"filter"`
```
This shouldn't be an array...